algod/run/run.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
```

Comment:
This script is probably where most of the effort went. All of the configuration and user facing functionality lives here.